README.md

```diff
@@ -29,33 +29,45 @@ Prerequisites
 Then clone the repo via git:
 
 ```bash
-$ git clone git@github.com:datopian/ckan-client-js.git
+$ npm install https://github.com/datopian/ckan-client-js.git
 ```
 
-And then install dependencies with npm.
-
-```bash
-$ cd ckan-client-js
-```
+or
 
 ```bash
-$ npm install
+$ yarn add https://github.com/datopian/ckan-client-js.git
 ```
 
 Now you can use the code:
 
 ```js
-const { Client } = require('./lib/index')
+const { Client } = require('ckanClient')
 ...
 ```
 
 ### Browser
 
-TODO
+If you want to use it as a script in the browser, then follow the steps below:
 
-```js
-import { Client } from "ckanClient";
-...
+```bash
+$ git clone git@github.com:datopian/ckan-client-js.git
+$ cd ckan-client-js
+$ npm install
+$ npm build
+```
+
+The last command will create `/dist/index.js` bundle which can be used in the browser like:
```

Comment:
IMO this is not good practice, because on any PR merged to master we have to run the build and push it, which I think for development workflow kinda overhead. So it's better to leave at users side and also then can build for different targets as well, such as RequireJS, CommonJS etc. Well technically they should touch webpack config in order to achieve that. If you want more easy option we can actually use https://unpkg.com/ to provide CDN. Basically every module published on NPM can be included as script from https://unpkg.com/ . In [frictionless-ckan-mapper-js](https://github.com/datopian/frictionless-ckan-mapper-js#use) we have the same approach.